docs/interactive.md

```diff
@@ -0,0 +1,128 @@
+# Interactive Python Console
+
+Tomviz provides a Python console and interactive API that can be used to build up
+and interact with pipelines from within Python. The Python console can be enabled
+using the View menu.
+
+
+# Loading a state file
+
+```python
+>>>
+Python 3.7.5 (default, Nov 20 2019, 09:21:52)
```

Comment:
I know you are just pasting the output, but I wonder if we should just omit the Python version/GCC version. It will be different for everyone, especially with future releases, and doesn't really document anything.